models/issue_list.go

```diff
@@ -422,13 +461,21 @@ func (issues IssueList) loadTotalTrackedTimes(e Engine) (err error) {
 			var totalTime totalTimesByIssue
 			err = rows.Scan(&totalTime)
 			if err != nil {
-				rows.Close()
+				// When there are no rows left and we try to close it, xorm will complain with an error.
+				// Since that is not relevant for us, we can safely ignore it.
+				if err := rows.Close(); err != nil {
+					log.Error("IssueList.loadTotalTrackedTimes: Close: %v", err)
+				}
 				return err
 			}
 			trackedTimes[totalTime.IssueID] = totalTime.Time
 		}
-		rows.Close()
-		left = left - limit
+		// When there are no rows left and we try to close it, xorm will complain with an error.
+		// Since that is not relevant for us, we can safely ignore it.
+		if err := rows.Close(); err != nil {
+			log.Error("IssueList.loadTotalTrackedTimes: Close: %v", err)
```

Comment:
This doesn't look like we're ignoring the empty rows.Close() error....

Reply:
Well, we're not really using it, only logging it. Do you think I should explicitly ignore it?

Reply:
(Also applies to the various other places where I use this)

Reply:
I think we should explicitly ignore it. Don't log things at error if they're not actually an error. Do you know is the error returned a reasonable type?

Reply:
Is this a bug of xorm of your comment?